include/circt/Dialect/FIRRTL/FIRAnnotations.h

```diff
@@ -0,0 +1,31 @@
+//===- FIRAnnotations.h - .fir to FIRRTL dialect parser -------------------===//
```

Comment:
Plz change the header comment line to be more specific.

Reply:
Fixed in the rebase.